proving_system/stark/src/cairo_vm/instruction_flags.rs

```diff
@@ -17,6 +19,26 @@ const OPCODE_MASK: u64 = 0x7000;
 const OPCODE_OFF: u64 = 12;
 const FLAGS_OFFSET: u64 = 48;
 
+// TODO: This is just an auxiliary function done to get out of the way.
+// It should be deleted afterwards
+#[allow(non_snake_case)]
+pub(crate) fn aux_get_last_nim_of_FE(value: &FE) -> u64 {
```

Comment:
We could get rid of this annotation renaming the function with something like this:
```suggestion
#[allow(non_snake_case)]
pub(crate) fn aux_get_last_nim_of_field_element(value: &FE) -> u64 {
```
or
```suggestion
#[allow(non_snake_case)]
pub(crate) fn aux_get_last_nim_of_felt(value: &FE) -> u64 {
```